package.json

```diff
@@ -11,6 +11,7 @@
     "start": "react-native start",
     "start:android": "react-native run-android --no-jetifier",
     "start:ios": "react-native run-ios",
+    "start:windows": "react-native start --use-react-native-windows",
```

Comment:
Mentioned somthing in another comment, but can we share the metro config and use `run-windows`?

Reply:
Not sure, but I think we still need to swap the metro config

Reply:
@acoates-ms did some work to allow a single metro config allowing multiple out of tree platforms for 0.62. Would you mind checking with him? Want to make sure we set the right patterns for future community modules based on 0.62.